src/Simulation.jl

```diff
@@ -228,7 +228,7 @@ function getconcentrationsensitivity(bsol::Simulation{Q,W,L,G}, numerator::Strin
     C = getC(bsol,t)
     c = bsol.sol(t)[indnum]/V
     k = bsol.domain.p[inddeno+length(bsol.domain.phase.species)]
-    val = k*(s-c*sum(svals)*R*bsol.domain.T/bsol.domain.P)/(c*V) #known T and P
+    val = k*(s-c*sum(svals)*R*getT(bsol,t)/bsol.domain.P)/(c*V) #known T and P
```

Comment:
Can you just use the T from further up in this function? and do the same for domain.P?